### PR TITLE
Develop <- 2DAngularController

### PR DESCRIPTION
**Pull request by Lizband**
Closes issue #9

### CODE REVIEW

**Review by labrys — CHANGES_REQUESTED** [review]
`invertHorizontalAxis` should be `invertRotationalAxis`

File: src/neverlab/Assets/Angular_Controller_2D.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    // Public Variables
+    //=-----------------=
+    public float speed;
+    [SerializeField] private bool invertHorizontalAxis;
+
+
+    //=-----------------=
+    // Private Variables
+    //=-----------------=
+    private Vector3 velocity;
```
Comment (labrys) [inline src/neverlab/Assets/Angular_Controller_2D.cs:17]: This should be a `Vector2` and `angularVelocity`

File: src/neverlab/Assets/Angular_Controller_2D.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Angular_Controller_2D : MonoBehaviour
+{
+    //=-----------------=
+    // Public Variables
+    //=-----------------=
+    public float speed;
+    [SerializeField] private bool invertHorizontalAxis;
```
Comment (labrys) [inline src/neverlab/Assets/Angular_Controller_2D.cs:11]: This should be `invertRotationalAxis`

File: src/neverlab/Assets/Angular_Controller_2D.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	    transform.Rotate(velocity * Time.deltaTime);
+    }
+
+    //=-----------------=
+    // Internal Functions
+    //=-----------------=
+    private void UserInput()
+    {
+	    // Horizontal
+	    if (Input.GetButton("TurnLeft")) velocity.z = speed;
```
Comment (labrys) [inline src/neverlab/Assets/Angular_Controller_2D.cs:48]: Maybe `RotateLeft` and `RotateRight` or `RotateClockwise` and `RotateCounterClockwise`

**Review by labrys — CHANGES_REQUESTED** [review]

File: src/neverlab/Assets/Angular_Controller_2D.cs (hunk 1 above)
Comment (Lizband) [reply src/neverlab/Assets/Angular_Controller_2D.cs:17]: Converting this to a Vector2 would prohibit access to the gameObject's Z axis (which is the one we would want a 2D object to rotate around by default)

Comment (labrys) [reply src/neverlab/Assets/Angular_Controller_2D.cs:17]: I'm pretty sure this shouldn't even be a vector for 2D and should only be an angle.

**Review by labrys — DISMISSED** [review]

**Review by labrys — APPROVED** [review]

**Pull request merged**